SocialFilter/sections/specific.txt

```diff
@@ -75,6 +75,7 @@ biomedya.com##.share-url
 pravmir.ru##.hh-share-block
 elhow.ru##.art-social-wrapper-ahchor
 elhow.ru##.social-banner
+sosyalhalisaha.com##p.social
```

Comment:
tag is not necessary here

Reply:
Please add the domain to existing rule
`##.social:not(body)`